src/workloads/scale/ContentionTTLDeletions.yml

```diff
@@ -14,35 +14,23 @@ Keywords:
 Clients:
   Default:
     QueryOptions:
-      maxPoolSize: 260
-
-NumPhases: &NumPhases 6
-
-Document: &generatedDoc
-  expireDate: {^Date: "1970-01-01"}  # Instantly expire the document
-  y: 999
-  b: "Arbitrary string with 45 different characters"
-
-ActorTemplates:
-- TemplateName: WritesActorTemplate
-  Config:
-    Name: {^Parameter: {Name: "Name", Default: WritesActor}}
-    Type: CrudActor
-    Database: &db test
-    Threads: {^Parameter: {Name: "Threads", Default: 1}}
-    Phases:
-      OnlyActiveInPhases:
-        Active: {^Parameter: {Name: "ActivePhases", Default: [1]}}
-        NopInPhasesUpTo: *NumPhases
-        PhaseConfig:
-          Duration: {^Parameter: {Name: "Duration", Default: 3 minutes}}
-          CollectionCount: &NumColls 1
-          Operations:
-          - OperationName: insertMany
-            OperationCommand:
-              Documents:  # Insert 1 document at a time
-              - *generatedDoc
+      maxPoolSize: 10000
+
+SmallDoc: &SmallDoc
+  a: {^FastRandomString: {length: 123}}
+  numField: {^RandomInt: {min: 0, max: 1000}}
+  counter: 0
+  expireDate: {^Date: "1970-01-01"}
+
+LargeDoc: &LargeDoc
+  a: {^FastRandomString: {length: 67890}}
+  numField: {^RandomInt: {min: 0, max: 1000}}
+  counter: 0
+  expireDate: {^Date: "1970-01-01"}
 
+WMajorityJFalse: &WMajorityJFalse
```

Comment:
This is the default setting in sys-perf now (on the variants suffixed with 2022-11), so I'm not sure that we need to include it here

Reply:
I don't think we need to test j:false